lib/requests/foods.js

```diff
@@ -5,10 +5,20 @@ const getFoods = () => {
     .catch(error => console.error({ error }))
 }
 
-const deleteFood = (id) => {
+const deleteFood = (id, event, removeFoodRow) => {
   fetch(`https://ivmh-qs-api.herokuapp.com/api/v1/foods/${id}`, {
     method: 'DELETE'
   })
+  .then(response => checkSuccessfulDelete(response, event, removeFoodRow) )
+}
+
+const checkSuccessfulDelete = (response, event, removeFoodRow) => {
+  if (response.ok) {
+    removeFoodRow(event)
+  } else {
+    var foodName = event.target.parentNode.parentNode.firstElementChild.textContent
+    alert(`This food is listed on at least one meal. You must remove "${foodName}" from all meals before deleting.`)
```

Comment:
This works for now to at least signify to the user as to why the row/food item cannot be deleted. It can be changed/styled to render in a more appealing manner.

Reply:
👍